tools/autograd/derivatives.yaml

```diff
@@ -1499,8 +1499,8 @@
   grad_output: grad.slice_symint(dim, start, end, step)
   result: auto_linear
 
-- name: slice_scatter(Tensor self, Tensor src, int dim=0, SymInt? start=None, SymInt? end=None, SymInt step=1) -> Tensor
-  self: slice_scatter_symint(grad, zeros_like(src), dim, start, end, step)
+- name: slice_inverse(Tensor(a) self, Tensor src, int dim=0, SymInt? start=None, SymInt? end=None, SymInt step=1) -> Tensor(a)
```

Comment:
nit: maybe add a comment here saying that `slice_scatter` inherits its derivative formula from `slice_inverse`, since as far as autograd the two ops are the same, but have different aliasing behavior in the forward.

Reply:
added!